README.rst

```diff
@@ -140,15 +140,19 @@ Click the bell icon at the top to view the My Notifications page (/notifications
 Automatic Scaling
 -----------------
 
-MetaCI can be configured to monitor its own build queue and scale its own Heroku dynos based on load. It will check the queue once a minute and add worker dynos when needed. Once all builds are complete, all worker dynos will be shut down. Heroku only bills for the dyno seconds used, so this scaling can save money while allowing for greater concurrency when desired.
+MetaCI can be configured to monitor its own build queue and scale its own Heroku dynos based on load in multiple Heroku Apps. It will check the queue once a minute and add worker dynos when needed. Once all builds are complete, all worker dynos will be shut down. Heroku only bills for the dyno seconds used, so this scaling can save money while allowing for greater concurrency when desired.
 
 To configure autoscaling:
 
 1. Set the METACI_MAX_WORKERS setting to the maximum number of dynos you'd like to scale up to.
 2. Set the METACI_WORKER_RESERVE setting to the number of dynos you'd like to reserve for high-priority builds. (Optional; defaults to 1.)
```

Comment:
@Br4nd0R reading this makes me think we should move the max workers and worker reserve settings to be part of the autoscaler config dict too. We don't necessarily want the same max for the robot app, and it doesn't need a reserve.